addon/components/power-select-with-create.js

```diff
@@ -12,11 +12,12 @@ export default Component.extend({
   matcher: defaultMatcher,
   suggestedOptionComponent: 'power-select-with-create/suggested-option',
   powerSelectComponentName: 'power-select',
+  searchEnabled: true,
```

Comment:
This is restoring the old default of `ember-power-select@^2.0.0`, isn't it? Shouldn't `<PowerSelectWithCreate>` have the same defaults as `<PowerSelect>`?

Reply:
:+1: That makes sense!